extensions/amp-consent/0.1/tcf-api-command-manager.js

```diff
@@ -293,12 +293,12 @@ export class TcfApiCommandManager {
       return false;
     }
     if (
-      parameter &&
-      command != TCF_POST_MESSAGE_API_COMMANDS.REMOVE_EVENT_LISTENER
+      (!parameter || isNaN(parameter)) &&
```

Comment:
I think you would want `parameter === null || isNaN(parameter)` only. isNaN will return true for `undefined`. In opposite, you would want to let `0` pass through, but `!parameter` would treat 0 as false.